src/AcmCountCard/AcmCountCard.stories.tsx

```diff
@@ -14,6 +14,10 @@ const handleCardClick = () => {
     alert('Card clicked')
 }
 
+const handleKeyPress = () => {
+    alert('Key pressed')
```

Comment:
Maybe print the key that was pressed?

Reply:
Changed to notify what key is pressed in the story